sbin/build.sh

```diff
@@ -101,7 +144,7 @@ configuringBootJDKConfigureParameter()
 
   echo "Boot dir set to ${JDK_BOOT_DIR}"
 
-  CONFIGURE_ARGS="${CONFIGURE_ARGS} --with-boot-jdk=${JDK_BOOT_DIR}"
+  addConfigureArg "--with-boot-jdk=" "${JDK_BOOT_DIR}"
```

Comment:
is the space between correct? `--with-boot-jdk=" "${JDK_BOOT_DIR}"`, I'm not sure that would pass through correctly?

Reply:
They are two separate arguments to the addConfigureArg function. They get concatenated with no space in the function. I have to do it split like that so we can check if a conflicting config parameter of the same name (but potentially different value) has been passed in by the user via the command line or environment variable.